tracim/tracim
# Fix/3696  servdev

See #3696.

## Checkpoints



**For developers**



**For code reviewers**


**For testers**


## Testing method

- login/out and verify the TLMs are properly transmitted
- launch applications in servdev mode, for example app file:
  - add `http://localhost:8075` to `development.ini/cors.access-control-allowed-origin`
  - run backend: `docker-compose up -d && pserve development.ini`
  - run `yarn run servdev`
  - go to http://localhost:8075
- check servdev apps are usable.
